lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
+    "amn't": "am not",
+    "'cause": "because",
+    "can't've": "cannot have",
+    "couldn't've": "could not have",
+    "daren't": "dare not",
+    "daresn't": "dare not",
+    "dasn't": "dare not",
+    "everyone's": "everyone is",
+    "gimme": "give me",
+    "gon't": "go not",
+    "hadn't've": "had not have",
+    "he've": "he have",
+    "he'll've": "he will have",
+    "he'd've": "he would have",
+    "here's": "here is",
+    "how're": "how are",
+    "how'd'y": "how do you",
```

Comment:
Two things: there's only one apostrophe, and I don't know where it goes because it's archaic. I live in howdy country and I've only ever written it "howdy"

Reply:
my best guess would be "howd'y" but agree, the usage now is "howdy" which would translate to "how do ye" or more colloquially "how do you do"

Reply:
Tbh I think it's "hi" much more often than it's "howdy" but it probably depends if you howdy from day to day and code switch to haidy a new acquaintance, or just howdy ironically and rarely haidy at all

Reply:
so is the proposed changes
- add "howdy" + "howd'y" aliases
- "how do you do"